vmdk_plugin/utils/test_util/VolumeCreationTestUtil.go

```diff
@@ -49,3 +55,136 @@ func DeleteVolume(name string, ip string) ([]byte, error) {
 func InvokeCommand(remoteHostIP string, cmd string) ([]byte, error) {
 	return exec.Command("/usr/bin/ssh", append(SSH_OPTS, "root@"+remoteHostIP, cmd)...).CombinedOutput()
 }
+
+// returns disk format of volume using Admin cli
+func GetDiskFormat_AdminCli(volName string, hostname string) string {
+	cmd := ADMIN_CLI_CMD + "-c volume,disk-format 2>/dev/null | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	result := strings.Fields(op)
+	diskFormat := result[1]
+	fmt.Println("disk format", diskFormat)
+	return string(diskFormat)
+}
+
+// returns disk format of volume using docker cli
+func GetDiskFormat_DockerCli(volName string, hostname string) string {
+	cmd := DOCKER_CLI_VOLM_CMD + volName
+	op := ExecCmd(hostname, cmd)
+	volProps := strings.Fields(op)
+	i := 0
+	for range volProps {
+		if volProps[i] == "\"diskformat\":" {
+			i = i + 1
+			break
+		}
+		i++
+	}
+	fmt.Println("disk format is  - ", volProps[i])
+	df := strings.Trim(volProps[i], "\"")
+	dskFormat := strings.Trim(df, ",")
+	result := strings.Trim(dskFormat, "\"")
```

Comment:
sorry I am able to follow the proposed code path .. this can be simplified using --format option (while invoking dkr vol inspect)